client/my-sites/post-relative-time-status/index.jsx

```diff
@@ -96,9 +97,21 @@ class PostRelativeTime extends React.PureComponent {
 			statusText = this.props.translate( 'pending review' );
 			statusClassName += ' is-pending';
 		} else if ( status === 'future' ) {
-			const scheduledTime = this.props.moment( this.props.post.date ).fromNow();
-
-			statusText = this.props.translate( 'scheduled %(scheduledTime)s', {
+			const moment = this.props.moment;
+			const scheduledDate = moment( this.props.post.date );
+			const now = moment();
+			const scheduledTime =
+				scheduledDate.diff( now, 'days' ) <= 7
+					? scheduledDate.calendar()
+					: scheduledDate.format( 'LL' ) +
+					  ' ' +
+					  this.props.translate( 'at', {
+							context: 'scheduled for 21 April at 02:00',
+					  } ) +
+					  ' ' +
+					  scheduledDate.format( 'LT' );
+
+			statusText = this.props.translate( 'scheduled for %(scheduledTime)s', {
```

Comment:
The translation comment isn't correct anymore.

Reply:
Oops, good catch! Fixed :)